src/terraform/commands.go

```diff
@@ -10,6 +10,9 @@ import (
 
 // RunInit runs the 'terraform init' command and force copies the remote state
 func RunInit(deployConfig types.DeployConfig) error {
+	if !util.DoesFileExist(deployConfig.TerraformFile) {
+		return errors.New("terraform initialization attempted in directory with no 'main.tf' file. Please make sure 'exo deploy build' has been run")
+	}
```

Comment:
At this point is running `exo deploy build` ever the wrong thing? If not can we just log something like `Missing 'main.tf' file. Initializing deployment...`